eth-connector/src/lib.rs

```diff
@@ -252,6 +253,20 @@ impl EthConnectorContract {
         true
     }
 
+    #[cfg(feature = "integration-test")]
+    #[result_serializer(borsh)]
+    #[must_use]
+    pub fn ft_balances_of(
```

Comment:
Because we are going to migrate accounts batches, so we need to get balances for multiple accounts at once to reduce the time that is needed for migration and gas usage.

Reply:
My question was not about that. This function exists only if the `integration-test` feature turns on. But why don't we just support this function in the `eth-connector` contract? Why do we need to create a fake one?

Reply:
This function returns mock values for tests and simulates the Aurora engine contract.
It is the same as the `verify_log_entry_in_bound` that simulates the Prover contract.